AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -764,11 +758,10 @@ - (void)transitionLayoutWithSizeRange:(ASSizeRange)constrainedSize
       }
       
       ASLayout *previousLayout = _layout;
-      [self _applyLayout:newLayout layoutTransition:nil];
+      [self setCalculatedLayout:newLayout];
```

Comment:
For me it seems inconsistence in two ways. We call it `calculatedLayout` thought our public API. There is no place where we call it only `layout`. As this is the case I actually think the right thing to do is to rename our instance variable from `_layout` to `_calculatedLayout`. Now we would be consistence throughout the class and could also call it `setCalculatedLayout:`.

Otherwise we can change it to `setLayout:` it makes totally sense in terms of setting an instance variable that's called `_layout`, but actually if you want to query it from the public API (the getter) we call it `calculatedLayout`. So the setter and getter don't have the same name than?? Calling the getter `- (ASLayout)layout` would not work  as we already have a method that is called `- (void)layout` but as you can see has a different return type and that's not possible in ObjC.

So to summarize I think the instance variable `_layout` is just a unfavorable name and we should rename that to `_calculatedLayout`?

@levi Does that make sense?

@nguyenhuy @appleguy @Adlai-Holler Do you guys have any opinion to rename the `_layout` instance variable to `_calculatedLayout` and add `setCalculatedLayout:` in ASDisplayNode